Recrutify/Recrutify.Host/Constants.cs

```diff
@@ -8,6 +8,7 @@ public static class Policies
             public const string FeedbackPolicy = nameof(FeedbackPolicy);
             public const string AdminPolicy = nameof(AdminPolicy);
             public const string HighAccessPolicy = nameof(HighAccessPolicy);
+            public const string ManagerPolicy = nameof(ManagerPolicy);
```

Comment:
you haven't registered policy in startup

Reply:
done